uber/models.py

```diff
@@ -552,6 +529,7 @@ def update_badge(self, attendee, old_badge_type, old_badge_num):
             :param old_badge_num: The old badge number.
             :return:
             """
+            from uber.badge_funcs import needs_badge_num
```

Comment:
i have no idea why I have to do this here, or why we're also doing a similar thing with check_ranges.  If I put this in line common.py, the entire world ends.  If I put it at the top of the file, unit tests explode.  For some reason it only works if this is right here.

It might be lack of sleep, but for now this is the best solution I have.